.github/workflows/ci.yml

```diff
@@ -35,12 +35,8 @@ jobs:
               python3 \
               python-is-python3 \
               libpython3-dev \
-              python3-dev \
-              python2-minimal \
-              python2-dev \
-              libpython2-dev
+              python3-dev
           elif [ "$RUNNER_OS" == "macOS" ]; then
-            brew install python@2
```

Comment:
I think you might still need a `brew install python` to get Python 3?

Reply:
I don't have enough experience with macOS. I'm not sure whether a python2 or the headers are installed. In the earlier versions it was not necessary to explicitly install python. Let's see what the macOS runner does.

Reply:
On my local macOS instal I have a system version of Python 2, but that will be removed in the future. The CI version of macOS that GitHub uses seems to come without a Python at all as far as I can tell.

Reply:
At the moment it looks good. The macOS runner just finished building gdb. Let's wait whether something else makes problems.

Reply:
Both runners (Ubuntu and macOS) finished successfully. So it seems that the patch worked and that we don't need an additional python installation.